src/game/default/g_types.h

```diff
@@ -482,8 +477,6 @@ typedef enum {
  * @brief Item types.
  */
 typedef enum {
-	ITEM_NONE,
-
```

Comment:
Hm. Are you sure we don't introduce ambiguity in removing these? Something with "item 0" is now ammo?